src/resources/app/pages/course/Assignment.vue

```diff
@@ -219,8 +225,13 @@ export default {
         this.uploadText = "Save"
       }
     },
+    overdue(dueDate) {
+      let now = new Date();
+      now = now.getFullYear() + `${(parseInt(now.getMonth()) + 1) > 9 ? "-" : "-0"}` + (parseInt(now.getMonth()) + 1) + "-" + now.getDate() + "T" + now.getHours() + ":" + now.getMinutes() + ":00Z";
+      return now > dueDate
+    },
     date(givenDate) {
-      return dateSlicing(givenDate);
+      return `${dateSlicing(givenDate)}` + " " + `${givenDate.slice(11,16)}`;
```

Comment:
```suggestion
      return `${dateSlicing(givenDate)} ${givenDate.slice(11,16)}`;
```
much simpler I guess